moz_telemetry/sandboxes/heka/analysis/moz_telemetry_new_experiment_monitor.lua

```diff
@@ -42,6 +42,31 @@ alerted = {
 
 new_experiments = {}
 
+-- Parse submission_text as JSON and return the value of "payload.testing".
+-- If it is not present, return false.
+function is_test(submission_text)
+    if not submission_text then
+        return false
+    end
+
+    local submission = cjson.decode(submission_text)
+    if not submission then
```

Comment:
The only way line 52 or 77 could fail this test is if null decoding is off and the JSON string is 'null' otherwise the previous line would have thrown an error.  The schema protects against this so it technically is not needed.